Cauldron/Assets/Scripts/UI/PotionSelect.cs

```diff
@@ -12,7 +12,7 @@ public class PotionSelect : GrowOnMouseEnter, IPointerClickHandler
         public bool clickable = true;
         public override void OnPointerClick(PointerEventData eventData)
         {
-            if (!clickable)
+            if (!clickable || book.isNightBook)
```

Comment:
Let's make this check a local property and use it in OnPointerEnter/Exit as well (that way we will indicate potion images as non-interactable).